internal/pkg/gateway/api_test.go

```diff
@@ -284,6 +292,20 @@ func TestGateway(t *testing.T) {
 				processProposalError: fmt.Errorf("mumbo-jumbo"),
 				errString:            "mumbo-jumbo",
 			},
+			{
+				name: "endorsement times out",
+				plan: endorsementPlan{
+					"g1": {"localhost:7051"},
+				},
+				signedProposal: validSignedProposal,
+				setupRegistry: func(reg *registry) {
+					reg.localEndorser.(*mocks.EndorserClient).ProcessProposalStub = func(context.Context, *peer.SignedProposal, ...grpc.CallOption) (*peer.ProposalResponse, error) {
+						time.Sleep(endorsementTimeout + 10*time.Millisecond)
```

Comment:
We should rarely see a sleep in tests because there's no way to preempt them.

Reply:
this test is now redundant - removed